docs/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.png
```

Comment:
This seems likely to bite us if we ever want to put images in. Is there a directory we can ignore instead?

Reply:
I have no idea what pngs we'd even want to ignore. I'll just remove this file completely and we can re-add it if building the docs locally creates files we want to ignore.